fabtests/configure.ac

```diff
@@ -61,6 +62,9 @@ AC_ARG_ENABLE([asan],
 AS_IF([test x"$enable_asan" != x"no"],
       [CFLAGS="-fsanitize=address $CFLAGS"])
 
+dnl Provider-specific tests set up
+FT_PROVIDER_TESTS_SETUP([efa])
```

Comment:
Why not always just build provider tests?

Reply:
Thanks, I removed the related change so always build provider tests.